src/pages/mypage/mypageReservationDetail/components/ClassContent/ClassContent.tsx

```diff
@@ -0,0 +1,41 @@
+import { useParams } from 'react-router-dom';
+import { useGetReservationsDetail } from '@/pages/mypage/mypageReservationDetail/apis/queries';
+import ClassInfo from '@/pages/reservation/components/ClassInfo/ClassInfo';
+import Text from '@/shared/components/Text/Text';
+import { sprinkles } from '@/shared/styles/sprinkles.css';
+import { getStatusMessage } from '@/shared/utils/getStatusMessage';
+import { getClassStatus } from '@/shared/utils/timeCalculate';
+
+const ClassContent = () => {
+  const lessonId = Number(useParams<{ id: string }>().id);
+
+  const { data } = useGetReservationsDetail(lessonId);
+
+  if (!data) {
+    return <div>오류 data 없음 </div>;
+  }
+
+  const lessonStartDateTime = data.rounds[0]?.startDateTime;
+  const lessonEndDateTime = data.rounds[0]?.endDateTime;
```

Comment:
해당 조건은 `data.rounds`가 반드시 배열이어야 한다는 가정이 존재합니다.
data.rounds가 undefined이거나 null이거나 배열이 아닌 경우엔 런타임 에러가 발생 합니다.
```suggestion
  const lessonStartDateTime = data.rounds?.[0]?.startDateTime;
  const lessonEndDateTime = data.rounds?.[0]?.endDateTime;
```

다음과 같이 수정하는 것이 data.rounds가 존재하지 않아도 에러 없이 undefined 반환하여서 더 안전할것 같네요.